File: python/TokoElektronik.py
class TokoElektronik:
    def __init__(self, nama_toko="", alamat_toko="", notel_toko=""):
        self.nama_toko = nama_toko
        self.alamat_toko = alamat_toko
        self.notel_toko = notel_toko

    # Getter & Setter
    def set_nama(self, nama_toko):
        self.nama_toko = nama_toko

    def get_nama(self):
        return self.nama_toko

    def set_alamat(self, alamat_toko):
        self.alamat_toko = alamat_toko

    def get_alamat(self):
        return self.alamat_toko

    def set_notel(self, notel_toko):
        self.notel_toko = notel_toko

    def get_notel(self):
        return self.notel_toko

    # Add data
    def add_data(self, nama_toko, alamat_toko, notel_toko):
        self.nama_toko = nama_toko
        self.alamat_toko = alamat_toko
        self.notel_toko = notel_toko

File: java/TokoElektronik.java
public class TokoElektronik {
    protected String namaToko;
    protected String alamatToko;
    protected String notelToko;

    public TokoElektronik() {}

    // Setter & Getter
    public void setNama(String namaToko) {
        this.namaToko = namaToko;
    }
    public String getNama() {
        return namaToko;
    }

    public void setAlamat(String alamatToko) {
        this.alamatToko = alamatToko;
    }
    public String getAlamat() {
        return alamatToko;
    }

    public void setNotel(String notelToko) {
        this.notelToko = notelToko;
    }
    public String getNotel() {
        return notelToko;
    }

    // Method add data
    public void addData(String namaToko, String alamatToko, String notelToko) {
        this.namaToko = namaToko;
        this.alamatToko = alamatToko;
        this.notelToko = notelToko;
    }
}
